demo/admin/src/common/ContentScopeProvider.tsx

```diff
@@ -37,10 +39,17 @@ export const ContentScopeProvider = ({ children }: Pick<ContentScopeProviderProp
 
     if (user.allowedContentScopes.length === 0) {
         return (
-            <>
-                Error: user does not have access to any scopes.
-                {user.impersonated && <StopImpersonationButton />}
-            </>
+            <MasterLayout>
+                <MainContent center>
```

Comment:
We could also think of introducing some special `Layout` Components.

- `CenterLayout`
- `FullPageLayout`
- `FillPageLayout``
- `StackLayout` -> Already available in MUI `Stack`
- `GridLayout` -> Already available in MUI `Grid`
- maybe others

Reply:
We already have [FullHeightContent](https://storybook.comet-dxp.com/?path=/docs/comet-admin-fullheightcontent--docs) and MainContent. I'm not sure if we should add this additional layouts via props or components. Having multiple components makes it harder to combine layouts (e.g., fill page and center), but the separation is clearer.

Reply:
`MainContent` is used for wrapping the main content of a page (basically adds padding)
Adding `fullHeight` combines the content-padding with the `FullHeightContent` behavior.

Reply:
These two examples would do the same, we just added `FullHeightContent` recently, as we didn't only need it for the main page content. 

```tsx
<MainContent fullHeight>
  // ...
</MainContent>
```

```tsx
<MainContent>
  <FullHeightContent>
    // ...
  </FullHeightContent>
</MainContent>
```